src/client/Client.cc

```diff
@@ -257,6 +257,10 @@ int Client::get_fd_inode(int fd, InodeRef *in) {
   return r;
 }
 
+bool Client::_ll_fh_exists(Fh *f) {
```

Comment:
wait -- aren't there any callers?

Reply:
yes the PR is incomplete, reason - waiting for this https://github.com/ceph/ceph/pull/55457#discussion_r1495318167 to come to conclusion. I'm inclined towards calling this helper in low level APIs instead of being called in the `Client::_preadv_pwritev_locked`, do let me know your thoughts.